Source/WebKit/UIProcess/mac/WebViewImpl.h

```diff
@@ -945,7 +947,9 @@ ALLOW_DEPRECATED_DECLARATIONS_END
     WeakObjCPtr<NSPopover> m_lastContextMenuTranslationPopover;
 #endif
 
+#if HAVE(REDESIGNED_TEXT_CURSOR)
     RetainPtr<NSObject> _textInputNotifications;
```

Comment:
should this be changed to _WKWebViewTextInputNotifications instead of NSObject?

Reply:
If you did, you could remove the dynamic downcast in ` WebViewImpl::updateCursorAccessoryPlacement`

Probably outside the scope of this PR, but up to you